modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/Rule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+/**
+ * Interface for a generic rule with the standard fields relevant for all policy rules.
+ */
+public interface Rule {
```

Comment:
A Ruleset has a set of rules and allows to define a version. That is to fulfil the OSO requirement to have versioned rules. But since there might be rules of different sources I entered the Ruleset to allow different sets, e.g., the open Antenna rules, the basic OSO rules, customer specific rules. All can be defined by different rulesets.